### PR TITLE
Add Java 23 support

### PR DESCRIPTION
**Pull request by pditommaso**
This PR adds the support for Java 23

### CODE REVIEW

File: docs/install.md [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 ## Requirements
 
-Nextflow can be used on any POSIX-compatible system (Linux, macOS, etc), and on Windows through [WSL](https://en.wikipedia.org/wiki/Windows_Subsystem_for_Linux). It requires Bash 3.2 (or later) and [Java 11 (or later, up to 22)](http://www.oracle.com/technetwork/java/javase/downloads/index.html) to be installed. You can see which version you have using the following command:
+Nextflow can be used on any POSIX-compatible system (Linux, macOS, etc), and on Windows through [WSL](https://en.wikipedia.org/wiki/Windows_Subsystem_for_Linux). It requires Bash 3.2 (or later) and [Java 11 (or later, up to 23)](http://www.oracle.com/technetwork/java/javase/downloads/index.html) to be installed. You can see which version you have using the following command:
```
Comment (christopher-hakkaart) [inline docs/install.md:9]:
```suggestion
Nextflow can be used on any POSIX-compatible system (Linux, macOS, etc), and on Windows through [WSL](https://en.wikipedia.org/wiki/Windows_Subsystem_for_Linux). It requires Bash 3.2 (or later) and [Java 11 (or later, up to 23)](http://www.oracle.com/technetwork/java/javase/downloads/index.html). You can see which version you have using the following command:
```

**Comment by vladsavelyev** [comment]
The installer at https://www.nextflow.io/docs/latest/install.html `curl -s https://get.nextflow.io | bash` still checks for Java 22 max

**Comment by pditommaso** [comment]
@vladsavelyev can your comment be considered resolved?

**Comment by vladsavelyev** [comment]
Yes!

**Comment by pditommaso** [comment]
🙌

**Pull request merged**